pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -795,7 +795,11 @@ cdef class _Timedelta(timedelta):
 
     @property
     def resolution(self):
-        """ return a string representing the lowest resolution that we have """
+        """
+        Return a string representing the lowest resolution that we have.
+        Note that this is nonstandard behavior.
```

Comment:
Instead, could you create a "See Also" section with the new method name, similar to this: https://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.isin.html